gems/grit/CVE-2013-4489.yml

```diff
@@ -0,0 +1,27 @@
+---
+gem: grit
+cve: 2013-4489
+ghsa: 95xq-v4m2-fq3r
+url: https://about.gitlab.com/releases/2013/11/04/gitlab-ce-6-2-and-5-4-security-release/
+title: GitLab Grit Gem for Ruby contains a flaw allowing
+  arbitrary commands to be executed
+date: 2022-05-17
+description: |
+  The Grit gem for Ruby, as used in GitLab 5.2 before 5.4.1 and
+  6.x before 6.2.3, allows remote authenticated users to execute
+  arbitrary commands, as demonstrated by the search box for the
+  GitLab code search feature.
+
+  GitLab Grit Gem for Ruby contains a flaw in the app/contexts/search_context.rb
+  script. The issue is triggered when input passed via the code
+  search box is not properly sanitized, which allows strings to
+  be evaluated by the shell. This may allow a remote attacker to
+  execute arbitrary commands.
+cvss_v2: 6.5
+patched_versions:
+  - ">= 2.6.1"
```

Comment:
I think this GHSA advisory is wrong. The [grit gem](https://rubygems.org/gems/grit) has no version 2.6.1. However, the [gitlab-grit gem](https://github.com/gems/gitlab-grit) gem does have a 2.6.1 version, and it's [ChangeLog mentions security fixes in 2.6.1](https://gitlab.com/gitlab-org/gitlab-grit/-/blob/v2.6.1/History.txt?ref_type=tags#L2). I've submitted a fix to the GHSA advisory.

Upon further investigation, it appears we already have a `gems/gitlab-grit/CVE-2013-4489.yml` file. So this is probably a duplicate.

Reply:
Delete as dup